src/main/java/com/google/cloud/spanner/pgadapter/statements/IntermediateStatement.java

```diff
@@ -47,43 +51,23 @@
   private static final char SINGLE_QUOTE = '\'';
 
   public IntermediateStatement(String sql, Connection connection) throws SQLException {
-    this();
     this.sql = sql;
+    this.sqlWithoutComments = PARSER.removeCommentsAndTrim(sql);
     this.statements = parseStatements(sql);
-    this.command = parseCommand(sql);
+    this.command = parseCommand(sqlWithoutComments);
     this.connection = connection;
     this.statement = connection.createStatement();
-  }
-
-  protected IntermediateStatement() {
-    this.executed = false;
-    this.exception = null;
-    this.resultType = null;
-    this.hasMoreData = false;
-    this.statementResult = null;
-    this.updateCount = null;
```

Comment:
Removed, as this is a no-op (it just sets a number of properties to their default values)